tests/unit/scheduler-test.js

```diff
@@ -25,223 +28,283 @@ module('Unit | Scheduler', function(hooks) {
     reset();
   });
 
-  test('whenRoutePainted resolves when transition ended', async function(assert) {
-    assert.expect(1);
+  [
+    {
+      moduleName: 'using PerformanceObserver',
+      capabilities: { performanceObserverEnabled: true },
+    },
+    {
+      moduleName: 'not using PerformanceObserver',
+      capabilities: { performanceObserverEnabled: true },
```

Comment:
Hmm, this seems odd. I would have expected:

```suggestion
      capabilities: { performanceObserverEnabled: false },
```

Reply:
Good catch! Tests still pass with this updated, thankfully.